spec/features/user_profile_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+
+feature 'User profile' do
+  given(:john) { Fabricate(:user, email: "john@example.com", password: "Password", name: "John Doe") }
+  given(:bob) { Fabricate(:user, email: "bob@codefish.org", password: "Test123", name: "Bob Hope") }
+  given!(:video1) { Fabricate(:video) }
+  given!(:video2) { Fabricate(:video) }
+  given!(:video3) { Fabricate(:video) }
+  given!(:queue_item1) { Fabricate(:queue_item, user: bob, video: video1) }
+  given!(:queue_item2) { Fabricate(:queue_item, user: bob, video: video2) }
+  given!(:queue_item3) { Fabricate(:queue_item, user: bob, video: video3) }
+  given!(:review1) { Fabricate(:review, video: video1, author: bob) }
+
+  background { log_in_user }
+
+  scenario "visit John's user profile page" do
```

Comment:
Give a description of what you are looking for: "User has an empty collection and no reviews", "User can see collection with reviews", etc.